### ISSUE TITLE
[NEW CHAPTER]- Ethical Decision Making

### ISSUE THREAD

**Issue by rockita** [alan-turing-institute/the-turing-way#552]
## Detailed description
 * This issue is a place to discuss matters relating to the writing of the chapter on _chapter_topic_.
 * This chapter is being written in _pull_request_number_
 * This chapter is being developed on the _chapter_topic_ branch.

## Current status
Chapter in the making

## Outline 

# What are research ethics?  (what the chapter is not about)
# What is different now
# Ethical decision making in the research data lifecycle

# A. Planning research
Design research
- Plan data management
- Plan consent for processing & sharing data
- Plan data collecting and processing protocols & templates
- Explore existing data sources

# B. Collecting data
- Collect data
- Capture data with metadata
- Acquire existing 3rd party data/data linkage

# C. Processing & Analysing
- Enter, digitize, transcribe, translate data
- Describe and document.
- Good data documentation includes information on:
  •At the data-level documentation
- Check, validate, clean
   • Data preprocessing
   • Dimensionality reduction
Anonymise
   • Anonymising quantitative data
    • Anonymising qualitative data
- Derive data
- Manage and store
- Analyze and interpret
    •Model training
    •Feature Provenance
    •Dataset partitions
    • Model provenance.
    •Model assessment:
    •Define acceptable model performance:
    • Check for bias/unfair outcomes (Diagnose, Handle_
- Randomisation control
- Produce research outputs
- Cite data sources

# D. Publishing and sharing data
- Establish copyright
- Create user documentation
- Data sharing agreements
- Create discovery metadata
- Select appropriate access to data
- publish/share data
- Promote data

# E. Preserve data
- Migrate data to best format
- Store and backup data
- Create preservation documentation
- Preserve & curate data

# F. Re-use of data
- Conduct secondary analysis
- Follow up research
- Conduct research reviews
- Scrutinize findings
- Use data for teaching & learning

# Outline
https://docs.google.com/document/d/1q7pehvcnB0LErqoHRF1oKnUeN_nkhclxj8m997-lrsc/edit#

## Updates
To avoid that others have to read through the full thread of comments, update the initial issue with important updates (e.g. decisions taken) regularly so that all the important information is available at one glance.

**Comment by pherterich** [comment]
There seems to be a fairly large overlap with the research data management chapter - so maybe you can have a look at https://the-turing-way.netlify.com/rdm/rdm.html and see what you can refer to already and safe yourselves some work not having to write all of the pieces.

**Comment by rockita** [comment]
thanks, @pherterich, good point. I've noticed- I will simplify my chapter

**Comment by pherterich** [comment]
You can also just add to the existing chapter where you feel is appropriate or split some of the existing content into your chapter that's covering all the ethics bit and then have your chapter be a bit like "all of the things in the RDM chapter apply, here's what you need to know in addition". Not sure what the best solution is, so feel free to add your contributions how you think they fit best.

**Comment by taunsquared** [comment]
It looks like you're writing about provenance as well! I'm writing a more high-level introduction to provenance in the chapter I suggested via issue #558, and I'm happy to link to other sections that also talk about provenance in more specific/technical details 😄

**Comment by rockita** [comment]
I'm updating the outline for this, as there were overlaps with other chapters. Roughly it will cover:


"A. Data Collection
 A.1 Informed consent: If there are human subjects, have they given informed consent, where subjects affirmatively opt-in and have a clear understanding of the data uses to which they consent?
 A.2 Collection bias: Have we considered sources of bias that could be introduced during data collection and survey design and taken steps to mitigate those?
 A.3 Limit PII exposure: Have we considered ways to minimize exposure of personally identifiable information (PII) for example through anonymization or not collecting information that isn't relevant for analysis?

B. Data Storage
 B.1 Data security: Do we have a plan to protect and secure data (e.g., encryption at rest and in transit, access controls on internal users and third parties, access logs, and up-to-date software)?
 B.2 Right to be forgotten: Do we have a mechanism through which an individual can request their personal information be removed?
 B.3 Data retention plan: Is there a schedule or plan to delete the data after it is no longer needed?

C. Analysis
 C.1 Missing perspectives: Have we sought to address blindspots in the analysis through engagement with relevant stakeholders (e.g., checking assumptions and discussing implications with affected communities and subject matter experts)?
 C.2 Dataset bias: Have we examined the data for possible sources of bias and taken steps to mitigate or address these biases (e.g., stereotype perpetuation, confirmation bias, imbalanced classes, or omitted confounding variables)?
 C.3 Honest representation: Are our visualizations, summary statistics, and reports designed to honestly represent the underlying data?
 C.4 Privacy in analysis: Have we ensured that data with PII are not used or displayed unless necessary for the analysis?
 C.5 Auditability: Is the process of generating the analysis well documented and reproducible if we discover issues in the future?

D. Modeling
 D.1 Proxy discrimination: Have we ensured that the model does not rely on variables or proxies for variables that are unfairly discriminatory?
 D.2 Fairness across groups: Have we tested model results for fairness with respect to different affected groups (e.g., tested for disparate error rates)?
 D.3 Metric selection: Have we considered the effects of optimizing for our defined metrics and considered additional metrics?
 D.4 Explainability: Can we explain in understandable terms a decision the model made in cases where a justification is needed?
 D.5 Communicate bias: Have we communicated the shortcomings, limitations, and biases of the model to relevant stakeholders in ways that can be generally understood?

E. Deployment
 E.1 Redress: Have we discussed with our organization a plan for response if users are harmed by the results (e.g., how does the data science team evaluate these cases and update analysis and models to prevent future harm)?
 E.2 Roll back: Is there a way to turn off or roll back the model in production if necessary?
 E.3 Concept drift: Do we test and monitor for concept drift to ensure the model remains fair over time?
 E.4 Unintended use: Have we taken steps to identify and prevent unintended uses and abuse of the model and do we have a plan to monitor these once the model is deployed?"
